pyproject.toml

```diff
@@ -38,6 +38,7 @@ requests = ">=2.21.0,<3.0"
 sentry-sdk = { version = ">=1.35.0,<2.0", optional = true }
 sqlalchemy = { version = ">=1.4.49,<2", optional = true }
 thrift-unofficial = ">=0.19.0,<1.0"
+typing-extensions = "^4.11.0"
```

Comment:
This is already a dependency, just an indirect one. Moved to direct since I import from it.